code/game/objects/items/crayons.dm

```diff
@@ -137,7 +137,7 @@
 	if(charges == -1)
 		. = FALSE
 	else if(!charges_left)
-		user << "<span class='warning'>There is no more of [src.name] \
+		user << "<span class='warning'>There is no more of the [src.name] \
```

Comment:
for future reference if you have a case like this where you're not sure if you need to use the or not just type /the and it does it automatically i believe
or maybe it's \the idk